README.md

```diff
@@ -37,18 +37,14 @@ Table of Contents
 
 # 1. Introduction
 
-This repository is a library for creating th2-data-services applications.
+This repository is an interface for creating th2-data-services applications.
 
-The library used to analyze stream data using _aggregate operations_ mainly from
-the ["Report Data Provider"](https://github.com/th2-net/th2-rpt-data-provider). Data Services allows you to manipulate
-the stream data processing workflow using _pipelining_.
+The library's features:
 
-The library allows you:
-
-- Natively connect to ["Report Data Provider"](https://github.com/th2-net/th2-rpt-data-provider) via
-  `ProviderDataSource` class and extract TH2 Events/Messages via _commands_
+- Provides core interface for developing data source implementations
```

Comment:
Return also 
`the stream data processing workflow using _pipelining_.`
we provide it by Data object